File: solutions/de.vimotest.sandbox/source_gen/de/vimotest/generationConfig/capitalizedGetterSetterNames/CapitalizedGetterNames_View.cpp

#include "CapitalizedGetterNames_View.h"
#include "CapitalizedGetterNames_View.h"
#include <vector>
#include <memory>
#include "CapitalizedGetterNames_ViewMyElementsRow.h"
#include <string>

bool CapitalizedGetterNames_View::GetIsMyFlagCheckBoxChecked( ) 
{
  return this->isMyFlagCheckBoxChecked;
}

bool CapitalizedGetterNames_View::GetMyVisibility( ) 
{
  return this->myVisibility;
}

bool CapitalizedGetterNames_View::GetIsMyFlagCheckBoxEnabled( ) 
{
  return this->isMyFlagCheckBoxEnabled;
}

std::string CapitalizedGetterNames_View::GetMyFlagCheckBoxText( ) 
{
  return this->MyFlagCheckBoxText;
}

std::vector<std::shared_ptr<CapitalizedGetterNames_ViewMyElementsRow>> CapitalizedGetterNames_View::GetMyRows( ) 
{
  return this->myRows;
}

File: solutions/de.vimotest.sandbox/source_gen/de/vimotest/generationConfig/capitalizedGetterSetterNames/CapitalizedGetterNames_View.cs
/// <filename>
///     CapitalizedGetterNames_View.cs
/// </filename>
using System.Collections.Generic;

public class CapitalizedGetterNames_View
{
    private bool isMyFlagCheckBoxChecked;
 
    public   virtual bool GetIsMyFlagCheckBoxChecked()
  {
    return this.isMyFlagCheckBoxChecked;
  }
 
    private bool myVisibility;
 
    public   virtual bool GetMyVisibility()
  {
    return this.myVisibility;
  }
 
    private bool isMyFlagCheckBoxEnabled;
 
    public   virtual bool GetIsMyFlagCheckBoxEnabled()
  {
    return this.isMyFlagCheckBoxEnabled;
  }
 
    private string MyFlagCheckBoxText;
 
    public   virtual string GetMyFlagCheckBoxText()
  {
    return this.MyFlagCheckBoxText;
  }
 
    private System.Collections.Generic.List<CapitalizedGetterNames_ViewMyElementsRow> myRows = new System.Collections.Generic.List<CapitalizedGetterNames_ViewMyElementsRow>();
 
    public   virtual System.Collections.Generic.List<CapitalizedGetterNames_ViewMyElementsRow> GetMyRows()
  {
    return this.myRows;
  }
}
